server/game/core/ability/abilityTargets/ArenaTargetResolver.ts

```diff
@@ -0,0 +1,54 @@
+import { TargetResolver } from './TargetResolver';
+import { IArenaTargetResolver } from '../../../TargetInterfaces';
+import { AbilityContext } from '../AbilityContext';
+import type Player from '../../Player';
+import PlayerOrCardAbility from '../PlayerOrCardAbility';
+import { Location } from '../../Constants';
+import { isArray } from 'underscore';
+import { isArena } from '../../utils/EnumHelpers';
+
+export class ArenaTargetResolver extends TargetResolver<IArenaTargetResolver<AbilityContext>> {
```

Comment:
I think it may be a bit overkill to create a whole target resolver for this situation, handful of reasons:

- Only two cards total that use it
- Doesn't align super well with the design of the other target resolvers in that it looks like a player or card resolver but the target "arena" isn't really an input to a system the same way that player or card is and therefore can't be handled implicitly in the same way
- I think the functionality is already pretty well covered by the select target resolver since it should just be two buttons each of which calls a system (see Leia unit or Reinforcement Walker for examples of use)